### PR TITLE
Inception Bar Done !

### PR DESCRIPTION
**Pull request by deadfishhackers**
Took me some time to do it and it was hard but VOila done! 

[Actions.zip](https://github.com/DarkSecDevelopers/HiddenEye/files/3211896/Actions.zip)

(i asked to upload my logo so dont bitch about it)
<hr>
<p align="center"></p>

### CODE REVIEW

**Comment by An0nUD4Y** [comment]
@deadfishhackers , Thanks buddy,
I will complete documentation whenever got time...!!!

And Welcome To the Team buddy..

**Comment by sTiKyt** [comment]
@An0nUD4Y why he can upload this logo?He made something useful already?

**Comment by sTiKyt** [comment]
Solve conflicts please!

**Comment by deadfishhackers** [comment]
@sTiKyt which conflicts? i didnt have any issues running it

**Comment by sTiKyt** [comment]
@deadfishhackers your version of HiddenEye were outdated, you have conflicts with almost EVERY file in this program.. Firstly you had to update your local version and then code new features

File: Defs/Actions.py [hunk 1]
```diff
@@ -7,395 +7,388 @@
...
 from urllib.request import urlopen
 from subprocess import check_output, CalledProcessError
-from sys import stdout, argv, exit
-from Defs.Configurations import readConfig, ifSettingsNotExists
-from Defs.Languages import *
+from sys import stdout, argv
+from Defs.Configurations import ConfigurationManager
+from Defs.Languages import LocalizationManager
 
 RED, WHITE, CYAN, GREEN, DEFAULT = '\033[91m', '\033[46m', '\033[36m', '\033[1;32m',  '\033[0m'
```
Comment (sTiKyt) [inline Defs/Actions.py]: This string were replaced by new color theme system, it does not exist anymore

File: HiddenEye.py [hunk 2]
```diff
@@ -5,36 +5,39 @@
...
-from Defs.Languages import *
+from Defs.Checks import PermissionsManager, ConnectionManager
+from Defs.Configurations import ConfigurationManager
+from Defs.Actions import InterfaceManager, ServerManager
+from Defs.Languages import LocalizationManager
+from Defs.ArgumentHandler import ArgumentHandler
 from os import system
 
+
 RED, WHITE, CYAN, GREEN, DEFAULT = '\033[91m', '\033[46m', '\033[36m', '\033[1;32m',  '\033[0m'
```
Comment (sTiKyt) [inline HiddenEye.py]: Again this outdated line..

**Comment by sTiKyt** [comment]
@deadfishhackers https://app.codacy.com/app/DarkSecDevelopers/HiddenEye/pullRequest?prid=3625579

**Comment by deadfishhackers** [comment]
Dude wtf do u mean anoneduy sent me that, I just touched Actions.py not
other files???

**Comment by An0nUD4Y** [comment]
@sTiKyt , Don't merge it if their are conflicts, i will work in it again but still the credit will goes to @deadfishhackers
I am busy , it will take nearly a week to do that...

**Comment by An0nUD4Y** [comment]
Guys , take a deep breath , I know why are these conflicts because the time when was worked at inception-bar and uploaded that script for further help , those latest commits were not present at that time , So worry not I will fix these conflicts too ,

**Comment by deadfishhackers** [comment]
@sTiKyt not my fault that it has conflicts as @An0nUD4Y said when he uploaded the inception-bar to bitly, it was outdated then, now stop ARGUEING with me.

File: HiddenEye.py (hunk 2 above)
Comment (deadfishhackers) [reply HiddenEye.py]: idk

**Comment by sTiKyt** [comment]
@deadfishhackers don't take it so close, i don't worry about arguing. All what i tell you - objective.
There is no problem with you but this Pull Request have them, i don't worry who made this really.
But if i see problems i contact author. So, why you are trying to make conflict out of true info?Don't be toxic, i'm just trying to help.

**Comment by An0nUD4Y** [comment]
Sorry , guys I will leave this project , I don't wanna to spend time with stupid argument...
As these are just mistakenly issues. Be mature , we are learning with each other

Thanks To Understand ,

Closing it here , @deadfishhackers , I will give you the credit whenever completed..

Please don't Reopen this PR , Thank You

**Comment by deadfishhackers** [comment]
U are the toxic kid here

**Comment by An0nUD4Y** [comment]
I have closed , please don't comment Here , create an issue if you have any issue/suggestions/ideas

**Pull request closed without merging**